src/quo/components/overlay/view.cljs

```diff
@@ -0,0 +1,25 @@
+(ns quo.components.overlay.view
+  (:require
+    [quo.components.overlay.style :as style]
+    [quo.foundations.colors :as colors]
+    [react-native.blur :as blur]
+    [react-native.core :as rn]))
+
+(defn view
+  [{:keys [type]} child]
```

Comment:
We could receive all the children and render it.

```suggestion
  [{:keys [type]} & children]
```
<br />

When using this component, this will help us to prevent wrapping of children with fragment `[:<>]` or view `[rn/view]` (to say it's a single child)

```clojure
;; receiving single-child
[overlay/view {:type :shell}
 [:<>
  [rn/view ...]
  [some-component ...]
  [some-other-component...]]]

;; receiving multiple children
[overlay/view {:type :shell}
  [rn/view ...]
  [some-component ...]
  [some-other-component ...]]

```

Reply:
ah i was looking for that, didn't know about `&` . thanks @smohamedjavid , example help a lot